Fortnox.NET.Tests/Fortnox.NET.Tests/SupplierInvoiceTests.cs

```diff
@@ -52,10 +54,26 @@ public async Task GetSupplierInvoicesFromDateToDateTest()
         [TestMethod]
         public async Task GetSupplierInvoice()
         {
+            var invoices = await GetSupplierInvoices();
+
+            if (!invoices.Data.Any())
+            {
+                return;
```

Comment:
Perhaps an Assert.Fail is better here as it should signal a failed test?

Reply:
I wouldn't say that the test failed, rather that there is nothing to test.

Reply:
I can see both sides here. Both failing and succeeding a test that has not been run is wrong IMO. If I had to chose, I'd say that false positives are worse than false negatives though.

My suggestion is to not check it at all, and let an exception happen if there are no invoices. That makes the most sence to me since it's not the test that is failing, but the prerequisits.

Reply:
I have opted to use Assert.Inconclusive. It should be a bit clearer than just letting the test die.

Reply:
Perfect! Great job Dylan! 💯